datanode/sqlstore/migrations/0099_add_stop_order_rejection_reason.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+
+ALTER TYPE stop_order_rejection_reason ADD VALUE IF NOT EXISTS 'REJECTION_REASON_STOP_ORDER_LINKED_PERCENTAGE_INVALID' AFTER 'REJECTION_REASON_STOP_ORDER_NOT_CLOSING_THE_POSITION';
```

Comment:
REJECTION_REASON_STOP_ORDER_NOT_CLOSING_THE_POSITION does seem to exist in the original migration 0027

Reply:
the order seems correct.

Reply:
yes, I'm adding the `REJECTION_REASON_STOP_ORDER_LINKED_PERCENTAGE_INVALID` value after the one that already exists, before 2 more values that were added with migration 0085. The order of the enum values has to match the sequence in the protos, so I have to be specific about where to add the value